cpp/src/games/stochastic_nim/main/UnitTests.cpp

```diff
@@ -14,6 +15,104 @@ using Types = Game::Types;
 using SymmetryGroup = groups::TrivialGroup;
 using GameResults = core::WinShareResults<Game::Constants::kNumPlayers>;
 
+class PerfectPlayerTest : public testing::Test {
+ protected:
+  using PerfectPlayer = stochastic_nim::PerfectPlayer;
+  using PerfectStrategy = stochastic_nim::PerfectStrategy;
+  using State = PerfectPlayer::base_t::State;
+  using ActionMask = PerfectPlayer::ActionMask;
+  using Params = PerfectStrategy::Params;
+  static constexpr core::seat_index_t Player0 = 0;
+  static constexpr core::seat_index_t Player1 = 1;
```

Comment:
We usually name constants like this `kPlayer0` instead of `Player0`.

With that said, it seems more natural to put these in `Constants.hpp`.